cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,19 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    const userName = 'Alex1205@gmail.com';
+    const password = 'kassandra01';
 
+    cy.visit('https://react-redux.realworld.io');
+
+    cy.get(':nth-child(2) > .nav-link')
+      .click();
+    cy.get(':nth-child(1) > .form-control')
+      .type(userName);
+    cy.get(':nth-child(2) > .form-control')
+      .type(password);
+    cy.get('.btn')
+      .click();
+    cy.get(':nth-child(4) > .nav-link').should('contain', userName);
```

Comment:
Changes to checking the presence of username in the navigation menu instead of email, as the username is expected to be displayed after login.